ApplicationLibCode/ProjectDataModel/WellLog/RimWellLogExtractionCurve.cpp

```diff
@@ -614,6 +614,36 @@ RiaDefines::PhaseType RimWellLogExtractionCurve::phaseType() const
     return phase;
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+QString RimWellLogExtractionCurve::wellDateFromGridCaseModel( RimCase* gridCaseModel, size_t timeStep )
+{
+    auto* geomCase    = dynamic_cast<RimGeoMechCase*>( gridCaseModel );
+    auto* eclipseCase = dynamic_cast<RimEclipseCase*>( gridCaseModel );
+
+    QStringList timeStepNames;
+
+    if ( eclipseCase )
+    {
+        if ( eclipseCase->eclipseCaseData() )
+        {
+            timeStepNames = eclipseCase->timeStepStrings();
+        }
+    }
+    else if ( geomCase )
+    {
+        if ( geomCase->geoMechData() )
+        {
+            timeStepNames = geomCase->timeStepStrings();
+        }
+    }
+
+    if ( timeStep >= 0 && timeStep < timeStepNames.size() ) return timeStepNames[timeStep];
+
+    return "01_Jan_2021";
```

Comment:
Change in behavior here: from empty date to "01_Jan_2021". Seems this will have to be changed occasionally?

Reply:
Maybe atleast update the year? QDate::currentDate().year();